src/test/java/org/opensearch/security/sanity/tests/SingleClusterSanityIT.java

```diff
@@ -34,6 +38,13 @@ public void testSecurityPluginInstallation() throws Exception {
         verifyPluginInstallationOnAllNodes();
     }
 
+    @Test
+    public void testAdminCredentials_customStrongPassword_shouldSucceed() throws Exception {
+        Response response = client().performRequest(new Request("GET", ""));
+        MatcherAssert.assertThat(response.getStatusLine().getStatusCode(), is(equalTo(200)));
+        MatcherAssert.assertThat(response.getStatusLine().getReasonPhrase(), is(equalTo("OK")));
```

Comment:
nit: use static import of `assertThat(...)`

Reply:
for some unknown reason, assertThat without inline import (with static import) always refers to JUnit assertThat instead of MatcherAssert's assertThat. That is why I had to resort to inline imports

Reply:
Aside: Can you dig into this outside of this PR? Having unknown behavior in codebase seems like a "Bad Thing" (tm)

Reply:
sounds good. Created an issue: https://github.com/opensearch-project/security/issues/3832